src/redirects.ts

```diff
@@ -0,0 +1,108 @@
+/**
+ * This file is part of Serlo.org Cloudflare Worker.
+ *
+ * Copyright (c) 2020 Serlo Education e.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @copyright Copyright (c) 2020 Serlo Education e.V.
+ * @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/serlo-org/serlo.org-cloudflare-worker for the canonical source repository
+ */
+
+import {
+  Url,
+  createNotFoundResponse,
+  Instance,
+  isInstance,
+  getPathInfo,
+} from './utils'
+
+export async function redirects(request: Request) {
+  const url = Url.fromRequest(request)
+
+  if (url.subdomain === 'start') {
+    return Response.redirect(
+      'https://docs.google.com/document/d/1qsgkXWNwC-mcgroyfqrQPkZyYqn7m1aimw2gwtDTmpM/',
+      301
+    )
+  }
+
+  if (url.subdomain === 'meet') {
+    if (url.pathname === '/') {
+      return Response.redirect('https://meet.google.com/vtk-ncrc-rdp')
+    } else if (url.pathname === '/dev') {
+      return Response.redirect('https://meet.google.com/rci-pize-jow')
+    } else if (url.pathname === '/einbindung') {
+      return Response.redirect('https://meet.google.com/qzv-ojgk-xqw')
+    } else if (url.pathname === '/begleitung') {
+      return Response.redirect('https://meet.google.com/kon-wdmt-yhb')
+    } else if (url.pathname === '/reviewing') {
+      return Response.redirect('https://meet.google.com/kon-wdmt-yhb')
+    } else if (url.pathname === '/labschool') {
+      return Response.redirect('https://meet.google.com/cvd-pame-zod')
+    } else {
+      return createNotFoundResponse()
+    }
+  }
```

Comment:
This is missing the new code of #108. You have to update the `master` and you need to merge it into your branch. Here you need to fix the merge conflict. See https://opensource.com/article/20/4/git-merge-conflict for more details. Please write me when you need help with this.